main_exercise7.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+from exercise7 import calculate_room_measurement_feet, convert_feet_to_meters
+
+def main():
+    """Get room dimensions and calcuate measurements in square feet and square meters"""
+
+    FEET_TO_METERS_CONVERSION = 0.09290304
```

Comment:
As I said in the previous file, this constant is only used there, so you should move it there.